src/java/winstone/AbstractSecuredConnectorFactory.java

```diff
@@ -0,0 +1,220 @@
+/*
+ * Copyright 2003-2006 Rick Knowles <winstone-devel at lists sourceforge net>
+ * Distributed under the terms of either:
+ * - the common development and distribution license (CDDL), v1.0; or
```

Comment:
it's the original license similar to others files and used for very long time. Or maybe I misunderstand your question?